clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -536,7 +535,12 @@ private void clearAddresses() {
         }
 
         public String toString() {
-            return "NodeState(" + state + ", " + lastConnectAttemptMs + ", " + failedAttempts + ", " + throttleUntilTimeMs + ")";
+            return "NodeState(" +
+                "state=" + state + ", " +
+                "lastConnectAttemptMs=" + lastConnectAttemptMs + ", " +
+                "failedAttempts=" + failedAttempts + ", " +
+                "failedConnectAttempts=" + failedConnectAttempts + ", " +
+                "throttleUntilTimeMs=" + throttleUntilTimeMs + ")";
```

Comment:
The original `toString` is unable to read because no leading variable name. Update it, and add `failedConnectAttempts`